Tools/samples.h

```diff
@@ -74,8 +74,8 @@ namespace AnaSamples
         }
 
         const T& null() const {return nullT_;}
-    protected:
         std::map<std::string, T> sampleSet_;
+    protected:
```

Comment:
I only needed the cross section from this class. Making it public enable me to do that.